backend/models.py

```diff
@@ -15,6 +15,13 @@ class User(db.Model):
     id = Column(Integer, autoincrement=True, primary_key=True)
     username = Column(String(200), unique=True)
```

Comment:
I'd keep this field.

Reply:
Actually username can work as github. Already existing records contains `username == github`. Therefore I will rename `user.username` to `user.github` and this will be used as login and won't add `user.github` at all?

Reply:
If we will use github auth I think it will be easier to remeber that `user.github` works as username, rather than `user.username` is actually github nick. So I would delete `user.username` altogether and keep just `user.github`. I think - let's not multiply entities that aren't necessary :)